src/server/game/Conditions/ConditionMgr.h

```diff
@@ -72,7 +72,8 @@ enum ConditionTypes
     CONDITION_ALIVE                 = 36,                   // 0                0              0                  true if unit is alive
     CONDITION_HP_VAL                = 37,                   // hpVal            ComparisonType 0                  true if unit's hp matches given value
     CONDITION_HP_PCT                = 38,                   // hpPct            ComparisonType 0                  true if unit's hp matches given pct
-    CONDITION_MAX                   = 39                    // MAX
+    CONDITION_REALM_ACHIEVEMENT     = 39,                   // achievement_id   instance_id    0                  true if realm achievement is complete
```

Comment:
and remove instance_id from documentation here, it is not a parameter for this condition type